### DISCUSSION TITLE
Don't show the "description" in the blog post

### DISCUSSION THREAD

**Discussion by benlindsay** [quarto-dev/quarto-cli#3495] [Q&A] [answered]
I like to have short summaries/descriptions of my blog posts show up in the home page listing all the blog posts, but I don't want them to show up in the blog posts themselves. I can't figure out how to configure that. For example, I have this at the top of a blog post's `index.qmd` file:

```qmd
---
title: "Analyzing Larger-than-Memory Data on your Laptop"
author: "Ben Lindsay"
date: "2017-03-10"
categories:
  - "python"
  - "recommender systems"
  - "dask"
  - "pandas"
  - "big data"
description: "Dask is an amazing Python library that lets you do all your Pandas-style dataframe manipulations with just a few simple tweaks so you don't have to worry about Jupyter freezing up."
---
```

Here's how that shows up in the post listings, as desired:

![image](https://user-images.githubusercontent.com/9361152/204440950-2ece95b3-7c24-4d38-8587-6da03789a98d.png)

But then in that post, I don't want that description (the highlighted part) cluttering up the head of the post

![image](https://user-images.githubusercontent.com/9361152/204441167-c21665fd-4323-420f-b6c1-636a55b5043f.png)

Is there a way to configure that so the summary shows up in the post listing but not in the post itself?

**Comment by mcanouil** [comment]
Quickly, my first thought would be to modify the `title-block.html` partial (<https://quarto.org/docs/journals/templates.html#html-partials>).

**Reply by benlindsay** [reply]
Thanks for this! So looking here:

https://github.com/quarto-dev/quarto-cli/blob/bf943eef0b1d42b178df26749800ed111629ac0d/src/resources/formats/html/templates/title-block.html#L18-L25

it looks like there's a "hide-description" variable that can hide the description, and looking here:

https://github.com/quarto-dev/quarto-cli/blob/1152cf92dd00df80cf63e64ca7f88c97de4a12c6/src/project/types/book/book-render.ts#L178-L179

it kinda looks like rendering to books makes use of that. So maybe a hook like that can be used for rendering websites too?

For now, I'm going with a simpler CSS hack that avoids dealing with the templates

**Reply by benlindsay** [reply]
@mcanouil Do you think it would make sense to create a new issue/feature request from this discussion? Happy to do that if it would be welcome.

**Reply by dragonstyle** [reply]
You are exactly correct, BTW- specifying `hide-description: true` in your document (or in a `_metadata.yml` for your posts folder) will prevent the display of the description. It looks like this slipped through the cracks as far as documentation / YAML validation. An issue would very much be appreciated - I will get it fixed for 1.3.

**Comment by benlindsay** [comment, marked as answer]
For my use case, adding this to the `styles.css` file did the trick:

```css
#quarto-document-content .description {
  display: none
}
```
